pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ak-vendor"
-version = "4.0.2"
+version = "4.0.3"
```

Comment:
Manual version update is not required, this is handled in deploy script https://github.com/appknox/vendor/blob/4e417bb9fe/scripts/deploy.sh#L11